/**:
  ros__parameters:
    connection_type: "tcp" # "tcp", "tty", or "udp"
    ip: "10.42.7.211" # Only used if connection_type is "tcp"
    port: 30200 # Only used if connection_type is "tcp" or "udp"
    device: "/dev/ttyUSB0" # Only used if connection_type is "tty"
    frame_id: "cg"
    debug: false

